externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/schema/RowSet.scala

```diff
@@ -377,4 +398,23 @@ object RowSet {
         other.toString
     }
   }
+
+  /** should stay in sync with [[org.apache.hadoop.hive.common.type.TimestampTZUtil]] */
+  var TIMESTAMP_LZT_FORMATTER: DateTimeFormatter = {
+    val builder = new DateTimeFormatterBuilder()
+    // Date part
+    builder.append(DateTimeFormatter.ofPattern("uuuu-MM-dd"))
```

Comment:
what does `uuuu-MM-dd` mean?

Reply:
The format is copied from TimestampTZUtil in `kyuubi-hive-jdbc` module. IIUC, `uuuu` has better compatibility with years before `1970`.